app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -186,5 +154,69 @@
         });
       }
     }
+
+    /*
+     * Determins if the item has damange and it's non kinetic.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
+     */
+    function itemHasNonKineticDamage(item) {
+      if (!item.dmg || item.dmg === 'kinetic') {
+        return false;
+      }
+      return true;
+    }
+
+    /*
+     * Check for a lockable or elemental property in an item.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
+     */
+    function asideHasLockOrElement(item) {
```

Comment:
Ah, interesting. I hadn't thought of this as an `<aside>` (which is generally used more for [sidebars, insets, etc.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside) in content, but hasn't really caught on in the web at large because in general "semantic markup" is a bit of a pipe dream).